eng/scripts/vcpkg.ps1

```diff
@@ -0,0 +1,33 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $Ref = (Get-Content "$PSScriptRoot/../vcpkg-commit.txt"),
```

Comment:
**Question:** Since we provide a `$Ref` and `$VcpkgPath` already, do these need to be optional parameter? Why not always require them?